### PR TITLE
update to use changes made for fluo-io/fluo#574

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: bin/impl/redeploy.sh [hunk 1]
```diff
@@ -67,6 +67,9 @@ if [ -n "$TARBALL" ]; then
   $SED "s#HADOOP_PREFIX=/path/to/hadoop#HADOOP_PREFIX=$HADOOP_PREFIX#g" $FLUO_HOME/conf/fluo-env.sh
 
   if [ $SETUP_METRICS = "true" ]; then
-    cp $FLUO_DEV/conf/fluo/metrics.yaml $FLUO_HOME/conf/
+    echo "io.fluo.metrics.reporter.graphite.enable=true" >> $FLUO_PROPS
```
Comment (mikewalch) [inline bin/impl/redeploy.sh]: What do you think about replacing the commented out default properties using sed rather than appending to fluo.properties?

Comment (keith-turner) [inline bin/impl/redeploy.sh]: that makes assumptions about whats in the config file.... could use sed to delete all lines containg  `io.fluo.metrics.reporter.graphite` before appending...

Comment (mikewalch) [inline bin/impl/redeploy.sh]: That's fine.  Leaving commented out properties can confuse people to think that the property is not set.

Comment (keith-turner) [inline bin/impl/redeploy.sh]: fixed in 36fa2a4

**Comment by mikewalch** [comment]
+1

**Pull request merged**